ios/ble/Utility/TransferHandler.swift

```diff
@@ -77,32 +77,35 @@ class TransferHandler {
 
     private func requestTransmissionReport() {
         var notifyObj: Data
-        delegate?.write(serviceUuid: BLEConstants.SERVICE_UUID, charUUID: NetworkCharNums.TRANSFER_REPORT_REQUEST_CHAR_UUID, data: withUnsafeBytes(of: 1.littleEndian) { Data($0) }, withResponse: true)
+        let data  = withUnsafeBytes(of: 1.littleEndian) { Data($0) }
```

Comment:
Was this fixed earlier using some other standard library call? I'll try to find the replacement code snippet for this logic